vms/event/tests/test_services.py

```diff
@@ -213,6 +217,63 @@ def test_check_edit_event(self):
         self.assertIn(j4.name, out4['invalid_jobs'])
         self.assertIn(j3.name, out4['invalid_jobs'])
 
+    def test_search_events(self):
+        """
+        tests search result for partial, exact and other searches on events
+        """
+        # if no search parameters are given,
+        # it returns all events
+        search_list = search_events("", "", "", "", "", "", "")
+
+        self.assertNotEqual(search_list, False)
```

Comment:
Why are you checking whether a list(empty/non-empty) is False or not?

Reply:
if the list is empty, the value will be False and it raises an exception

Reply:
checked it's implementation. Seems fine.